rust/Dockerfile

```diff
@@ -46,7 +46,7 @@ CMD ${PACKAGE}
 # Build an image for GitHub Actions which includes debug asserts and more test utilities
 FROM runtime AS debug
 
-RUN apk add --no-cache iperf3 bind-tools iproute2 jq procps iptables
+RUN apk add --no-cache iperf3 bind-tools iproute2 jq procps iptables ethtool
```

Comment:
Not necessarily against having `ethtool` in here but just curious if this was being actively needed in here?